products/sle/main.pm

```diff
@@ -914,6 +914,7 @@ sub load_fips_tests_misc() {
     loadtest "console/journald_fss.pm";
     loadtest "fips/curl_fips_rc4_seed.pm";
     loadtest "console/consoletest_finish.pm";
+    loadtest "fips/git_fips.pm";
```

Comment:
as there is nothing fips specific in your test, can you just make that a console test and also load it in opensuse and sle's extra test?